<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account_management</class>
 <widget class="QMainWindow" name="Account_management">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account_management</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>341</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Date Of Birth</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Full Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>ID number</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="name"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="mailbox"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ID"/>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="dateEdit">
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Occupation</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="occu"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="Register">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>280</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QListWidget" name="Username_list">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>251</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Search_Box">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="search">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>13</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="option">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>60</y>
        <width>57</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>User</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="user">
      <property name="geometry">
       <rect>
        <x>373</x>
        <y>60</y>
        <width>186</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="remove_box">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>301</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="remove">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>191</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="acc_list">
       <property name="geometry">
        <rect>
         <x>28</x>
         <y>35</y>
         <width>251</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="add_box">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>301</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>251</width>
         <height>162</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="2">
         <widget class="QCheckBox" name="user_2">
          <property name="text">
           <string>User</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Account</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="pass"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="Account"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="sysadmin">
          <property name="text">
           <string>SysAdmin</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="lib">
          <property name="text">
           <string>Librarian</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Role</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="add">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>190</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Remove_u">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>150</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Remove User</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="statusbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>Search_Box</tabstop>
  <tabstop>search</tabstop>
  <tabstop>Username_list</tabstop>
  <tabstop>user</tabstop>
  <tabstop>acc_list</tabstop>
  <tabstop>Account</tabstop>
  <tabstop>sysadmin</tabstop>
  <tabstop>lib</tabstop>
  <tabstop>user_2</tabstop>
  <tabstop>pass</tabstop>
  <tabstop>add</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>option</tabstop>
  <tabstop>name</tabstop>
  <tabstop>mailbox</tabstop>
  <tabstop>ID</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>occu</tabstop>
  <tabstop>Register</tabstop>
  <tabstop>Remove_u</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
